### DISCUSSION TITLE
How do I conrtribute on sourcehut?

### DISCUSSION THREAD

**Discussion by unbeatable-101** [tenacityteam/tenacity-legacy#160] [Q&A] [answered]
PRs aren’t a core git feature, and sourcehut doesn’t have its own version of them (That I can find), so how can I submit a patch for issue #146?

**Comment by Semisol** [comment, marked as answer]
https://git-send-email.io/ for an interactive demo

Or do the following:
1. Make sure you have SMTP access to the mail account your Git instance uses
2. Configure SMTP in git using the instructions at https://git-send-email.io/#step-2 (don't click next, just follow this)
3. Clone the repo
4. Add the patch
5. Commit it
6. Send it over `git send-email --to="~tenacity/tenacity-dev@lists.sr.ht" HEAD^` (to send the last commit, to send say the last N commits replace `HEAD^` with `HEAD~N`)
7. If you get any requests for changes, do the same command again but append `-v2` (increase it for every update)
8. After it gets accepted, enjoy

**Reply by fossdd** [reply]
And here is the git link to clone: `https://git.sr.ht/~tenacity/tenacityaudio.org`
